gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/mapper/CodeableConceptMapper.java

```diff
@@ -205,28 +206,23 @@ private CodeableConcept generateCodeableConceptUsingConceptId(CD mainCode, CD tr
     }
 
     private void addNonSnomedCodesToCodeableConcept(CodeableConcept codeableConcept, CD codedData) {
```

Comment:
I got a bit confused looking at this code, but I think my confusion stems from the name.

This method appears to populate the Coding block for all codes, snomed and non-snomed. Am I correct in that understanding? Perhaps the method should be called something like `populateCodableConceptCodings`?

Reply:
I'm happy to take a look at refactoring it, it is pretty complicated in the way the snomed code / extension is built, but I'm sure it can be simpler